.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       run: |
         mkdir build
         cd build
-        cmake .. -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1h/ -DOPENSSL_LIBRARIES=/usr/local/Cellar/openssl@1.1/1.1.1h/lib/
+        cmake .. -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1j/ -DOPENSSL_LIBRARIES=/usr/local/Cellar/openssl@1.1/1.1.1h/lib/
```

Comment:
any particular version for bumping the openssl version that you're testing with here? I also notice in the second argument here is still pointing to the "h" release instead of j, so it looks like a mismatch.

Reply:
openssl is part of the macOS virtual environment provided by GitHub so it has the latest version of openssl 1.1, for linux it's still under version 1.1 but on mac it has the full version. If I don't the build would fail.

I should've updated both of them though, so I'll make that change.

Reply:
This worked for me.  I have a mac as well.